src/app/body/quick-notes/addnew-note/addnew-note.component.ts

```diff
@@ -25,17 +26,26 @@ import { ToolsService } from 'src/app/services/tool/tools.service';
 export class AddnewNoteComponent implements OnInit {
 
   @Output() addNoteCompleted = new EventEmitter<boolean>();
+  @Output() showList = new EventEmitter();
```

Comment:
In which parent component are we using this event emitter.

Reply:
Line Removed.